ansible-chatbot-deploy.yaml

```diff
@@ -58,6 +58,13 @@ spec:
         volumeMounts:
           - name: ansible-chatbot-storage
             mountPath: /.llama/data
+        command:
```

Comment:
@TamiTakamiya The solution of the symlink works for me, but this code have 2 problems:
- It only works on k8s, not for local, maybe better creating again an entrypoint.sh and adding all the commands once buliding the image, so it will run everywhere
- This ansible-chatbot-deploy.yaml file will dissappear because of we'll move the deployment manifests to the -ops repo, see https://github.com/ansible/ansible-wisdom-ops/pull/1386

Reply:
I agree about moving this _large_ command to a script file; it's getting a bit of a beast being inline.

Reply:
thanks guys!! <3